src/main/java/com/projects/StartFinder.java

```diff
@@ -0,0 +1,37 @@
+package com.projects;
+
+import lombok.Getter;
+import lombok.Setter;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+@Command(name = "Index finder", description = "Finds indexes of List items where a number is present.")
+public class StartFinder implements Runnable {
+
+    @Getter
+    private ExecutorService searchExecutor;
+    @Getter
+    private StorageManager storageManager;
+    @Getter
+    private ThreadManager threadManager;
+
+    @Setter
+    @Option(names = {"-s", "--search"}, defaultValue = "0", description = "A number which need to search.", interactive = true)
+    private int searchNumber;
+    @Setter
+    @Option(names = {"-p", "--poolSize"}, defaultValue = "10", description = "Count of threads in pool.", interactive = true)
+    private int poolSize;
+
+    @Override
+    public void run() {
+        searchExecutor = Executors.newFixedThreadPool(poolSize);
```

Comment:
maybe set by constructor

Reply:
no, because we don't have arguments when constructor starts